src/updaters/composite.ts

```diff
@@ -40,3 +40,27 @@ export class CompositeUpdater implements Updater {
     return content || '';
   }
 }
+
+export function mergeUpdates(updates: Update[]): Update[] {
+  const updatesByPath: Record<string, Update[]> = {};
```

Comment:
Would it make sense to have `Record<Update['path'], Update[]>`?

Reply:
That would probably work in case update's path's type changes, but I think for readability, `string` might be easier to read.